_dashboards/im-dashboards/notifications.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: Notification settings
+parent: Index management in Dashboards
+nav_order: 60
+---
+
+# Notification settings
+
+You can configure default notification settings for index operations globally on the **Notification settings** page. If you need to override any of the settings, you can also configure notification settings for individual index operations.
```

Comment:
configuring notification settings for individual index operation will not override global setting, it is an additional setting. prefer to say `If you need to add additional settings`